tools/rpau/src/rpau/__init__.py

```
@@ -0,0 +1,75 @@
+import functools
+from concurrent.futures import ProcessPoolExecutor
+from typing import TYPE_CHECKING
+
+import tomllib
+
+from rpau.cli import build_argparse
+from rpau.logger import build_root_logger, get_logger
```

Flake8 (7.2.0)

[error] 'rpau.logger.get_logger' imported but unused

(F401)